# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: ipc/glue/PBackground.ipdl
# Commit: 5e078f860dfb
# Full Hash: 5e078f860dfb9368bd747e4a8c53d07c7e8809ee
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-13 04:28:43
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 2 - CookieStore API - IPDL, r=edgul,webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D215145
# ==============================================================================

diff -r d55b6525d7b2 -r 5e078f860dfb ipc/glue/PBackground.ipdl
--- a/ipc/glue/PBackground.ipdl	Thu Sep 12 16:48:18 2024 +0000
+++ b/ipc/glue/PBackground.ipdl	Thu Sep 12 16:48:19 2024 +0000
@@ -20,6 +20,7 @@
 include protocol PCacheStorage;
 include protocol PCacheStreamControl;
 include protocol PClientManager;
+include protocol PCookieStore;
 include protocol PEndpointForReport;
 include protocol PFileSystemManager;
 include protocol PFileSystemRequest;
@@ -99,6 +100,7 @@
   manages PCacheStorage;
   manages PCacheStreamControl;
   manages PClientManager;
+  manages PCookieStore;
   manages PEndpointForReport;
   manages PFileSystemRequest;
   manages PGamepadEventChannel;
@@ -197,6 +199,8 @@
   async PUDPSocket(PrincipalInfo? pInfo, nsCString filter);
   async PBroadcastChannel(PrincipalInfo pInfo, nsCString origin, nsString channel);
 
+  async PCookieStore();
+
   async PServiceWorkerManager();
 
   async ShutdownServiceWorkerRegistrar();